addons/website_event_booth/controllers/event_booth.py

```diff
@@ -18,9 +18,13 @@ def event_booth_main(self, event, booth_category_id=False, booth_ids=False):
             raise Forbidden()
 
         booth_category_id = int(booth_category_id) if booth_category_id else False
+        menu = request.env["website.menu"].search([('url', '=', request.httprequest.path)])
+        view = request.env["website.event.menu"].sudo().search([('event_id', '=', event.id), ('menu_id', '=', menu.id)]).view_id
```

Comment:
Don't think we need to do all this, we know we're trying to access the booth submenu, and we know the event. So just `event.booth_menu_ids.view_id` should work fine

Reply:
It works fine everywhere except for the tracks as, `track_menu_ids` has 2 menus: `track` and `agenda`. So, tried something different there.